Electron/src/Core/components/Paint.js

```diff
@@ -0,0 +1,257 @@
+import ReactDOM from "react-dom"
+import THREE from "./Three"
+
```

Comment:
Surprisingly, I don't think so. This file is copied from the STL-Viewer module and slightly adjusted to create props for rotation and zoom. It could be updated to .tsx if necessary, but it is a large file that I didn't write, so it wouldn't be very easy to do. However, these files should probably be cleaned up at some point